modules/autopairs/nvim-autopairs/nvim-autopairs.nix

```diff
@@ -12,7 +17,7 @@ in {
         description = "Set the autopairs type. Options: nvim-autopairs [nvim-autopairs]";
       };
 
-      nvim-compe = {
+      nvim-compe.setupOpts = lib.nvim.types.mkPluginSetupOption "nvim-compe" {
```

Comment:
Unnecessary setup options? The plugin has been removed in flake inputs and there is no configuration.